### DISCUSSION TITLE
Tamanho/proporção da caixa delimitadora na visualização AR.

### DISCUSSION THREAD

**Discussion by FellipeXavier** [google/model-viewer#2086] [Q&A]
olá, 

primeiramente gostaria de agradecer por essa ferramenta incrível (model-viewer).

pesquisei bastantes mas não encontrei uma forma de alterar a caixa delimitadora na visualização em realidade aumentada.

Aqui trago dois exemplos para tentar esclarecer minha dúvida:

1 foto com objeto de comprimento de 0,10 metros, e o outro objeto de 1,00 metros (não estão na escala original, mas não interfere na dúvida). 

![WhatsApp Image 2021-03-03 at 15 11 39](https://user-images.githubusercontent.com/67978896/109852291-65eae000-7c33-11eb-9476-29c25c3f3ce2.jpeg)
![WhatsApp Image 2021-03-03 at 15 11 26](https://user-images.githubusercontent.com/67978896/109852292-66837680-7c33-11eb-8661-abbb4c160094.jpeg)

Minha dúvida, é que a caixa delimitadora em branco ao redor do objeto ( que apaga quando o objeto não esta sendo manobrado) tem um tamanho padrão independente do tamanho do objeto. É possível alterar as dimensões desta caixa delimitadora, ( o que faria melhorar o manejo do objeto com relação a movimentar, rotacionar e alterar a escala) para que fique proporcional ao objeto, ou até mesmo que fique em um tamanho fixo colocado manualmente?

Agradeço a atenção.

**Comment by FellipeXavier** [comment]
Hello,

first I would like to thank you for this amazing tool (model-viewer).

I searched a lot but I couldn't find a way to change the bounding box in the augmented reality view.

Here I bring two examples to try to clarify my doubts:

1 photo with an object of length of 0.10 meters, and the other object of 1.00 meters (they are not in the original scale, but it does not interfere in the doubt).

WhatsApp Image 2021-03-03 at 15 11 39
WhatsApp Image 2021-03-03 at 15 11 26

My doubt is that the blank bounding box around the object (which goes out when the object is not being maneuvered) has a standard size regardless of the size of the object. Is it possible to change the dimensions of this bounding box, (which would improve the handling of the object in relation to moving, rotating and changing the scale) so that it is proportional to the object, or even that it is in a fixed size placed manually?

I appreciate the attention.

**Comment by FellipeXavier** [comment]
@elalish Lalish , do you know if you can place a proportional marker in the augmented reality view? Thanks

**Reply by elalish** [reply]
No, there isn't control for that currently. However, we do want to improve our AR UX in a number of ways, so we'll keep this in mind. It's currently sized by the bounding box of the model plus a fixed amount in order to ensure that tiny objects can still be found. It's also designed more for floor height than desk height.

**Reply by FellipeXavier** [reply]
got it, thank you very much for the clarification.
